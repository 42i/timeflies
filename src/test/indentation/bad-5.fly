wp a
    aa
        aaa
        bbb
        ccc
    bb
        aaa
        	aaaa
        	bbbb
        bbb
            aaaa
            bbbb
        ccc
    cc
