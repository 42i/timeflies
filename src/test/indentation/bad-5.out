indentation/bad-5.fly:8: ERROR : indentation contains both spaces and tabs
indentation/bad-5.fly:9: ERROR : indentation contains both spaces and tabs
